assets/src/edit-story/app/highlights/styles.js

```diff
@@ -35,6 +35,5 @@ export const FLASH = css`
 `;
 
 export const OUTLINE = css`
-  box-shadow: 0 0 0 2px ${dsTheme.colors.border.focus};
-  border-radius: ${dsTheme.borders.radius.small};
+  ${({ theme }) => themeHelpers.focusCSS(theme.colors.border.focus)};
```

Comment:
I think we should keep the border-radius here, it gets lost on things like the story title since that border isn't shown unless in an active state. And since the border.focus is the default you don't actually need to pass a color in here at all, you could just do `${themeHelpers.focusCss}`